plugins/PostProcessingPlugin/Script.py

```diff
@@ -22,6 +22,24 @@
 if TYPE_CHECKING:
     from UM.Settings.Interfaces import DefinitionContainerInterface
 
+# Some post-processing tasks are easy to parallelize. The easiest way to parallelize python code is via multiprocessing.
+# Post-processing scripts aren't directly importable, though,  making them incompatible with multiprocessing.
+# This module is importable, so it's possible to provide a workaround. The trick is to set up functions to call
+# before multiprocessing forks the main process. That way, each forked process has a copy of the processing script.
+# For example:
+#         pool = multiprocessing.Pool(processes=multiprocessing.cpu_count(), initializer=multiprocessing_init,
+#                                     initargs=({"process_layer": self.process_layer},))
+#        result = pool.apply_async(multiprocessing_call, ("process_layer", (layer_steps,))))
+#        return result.get()
+def multiprocessing_init(functions_by_name):
+    """ An inititializer for multiprocessing pools. The passed in keyword arguments are callable using the
+        multiprocessing_call function below """
```

Comment:
Please put function and class documentation in Doxygen-style comments above the function as well. For now at least.

You can use `\param` and such to denote parameters.